### DISCUSSION TITLE
documentation

### DISCUSSION THREAD

**Discussion by ringo458** [thliebig/openEMS-Project#32] [General]
Hi,
What I see on github is a collection of tex files. Is that the source of the web documentation? I'm not sure how to process that into a document that I can keep around and look at or print. I pulled down a latex console but could not get anywhere with it. I'm assuming that to jump start getting anything done I just have to rely on interpolating around the examples. If I'm missing something and there is a master document around please let me know. All I can see is some documentation for python versions of the CSXCAD.

**Comment by thliebig** [comment]
Hi, no the tex docu is really ancient? Currently I'm working on a rst docu. The current version you can find [here](https://docs.openems.de/).
Any help is highly appreciated.. The source code of the doc is in this repository...

**Comment by biergaizi** [comment]
For many years, openEMS used to have a wiki with much more detailed information than what's available here. Unfortunately, it's currently offline, it's my understand that they will never go back online again - my understanding of the situation is that the server hosted at University of Duisburg-Essen had crashed in late <del>2021</del> 2022 and everything was lost.

For now, you can refer to the documentation on the wiki preserved by archive.org:

https://web.archive.org/web/20211103161807/http://openems.de/index.php/Tutorials.html
https://web.archive.org/web/20211023143824/https://openems.de/index.php/Overview.html
https://web.archive.org/web/20211023152815/http://openems.de/index.php/Introduction.html
https://web.archive.org/web/20211023145021/https://openems.de/index.php/Frequently_Asked_Questions.html

In my opinion, everything written here should be copied to GitHub, this should be a priority of the project in <del>2022</del> 2023.

**Reply by thliebig** [reply]
If you add +1 to all your years I fully agree ;)
I have the wiki still after I converted it to a static version. But I'm not sure if I should upload it anywhere. I would really like to get this sorted and put into the github docu.

Any help here would be very much appreciated as my spare time is really limited...

**Reply by ringo458** [reply]
I'd consider helping but I don't know shit about git. Seriously though I understand that github is just a website but if there is any requirement to understand the underlying metaphor I would fall short. What I might feel more comfortable about is providing an example related to something that I happen to be working on. 

About the documentation, I would wget and then do an amateurish job of putting those html documents on my own currently empty github page as a project. I did not even know that Wikis are not always hosted by Wikimedia. The ins and outs of Wiki are as opaque to me as they are to  most people but I think that the html should be saved at least to github and I should be able to do that. But openEMS ought to be back on Wiki don't you think? I have some hosting infrastructure that I am not using but all my skills are pretty dated. 

LMK if I can do any of the above. I am a semi-retired system integrator who wrote large quantities of scientific C back in the 90's but never moved past that coding stage to the current levels of abstraction and sophistication.

**Reply by thliebig** [reply]
I still had the hope that the university would get their stuff back up and running at some point... But I really have no clue. At the moment they are still mostly offline...
I could of course create yet another github repository for the wiki page and see if I can host it this way... 
I'm even a bit more sad about the forum... we had so many great questions there, but I have no static version of it and grabbing it from the web-archives did not give me a really hostable version either...

**Comment by thliebig** [comment]
I have created a new repo hosting the legacy html [wiki](https://github.com/thliebig/openEMS-wiki).

You can find it now here: [https://wiki.openems.de](https://wiki.openems.de/)  

Please give it a try. A bit of cleanup should still be made? Any help is welcome!!!

Let me know what you think.

**Reply by thliebig** [reply]
BTW. it is not yet linked by the "main" openEMS.de website
